### PR TITLE
Fix saving problem

### PR DESCRIPTION
**Pull request by rexagod**
References (and fixes) #38 #37

### CODE REVIEW

**Comment by rexagod** [comment]
@jywarren Please review.

**Comment by jywarren** [comment]
Ah, i see, cool!

File: dist/inlineMarkdownEditor.js [hunk 1]
```diff
@@ -10301,7 +10299,7 @@ module.exports = function isEditable(markdown, originalMarkdown) {
 },{}],99:[function(require,module,exports){
 module.exports = function onComplete(response, markdown, html, el, uniqueId, form, o) {
   var message = form.find('.section-message');
-  if (response === 'true' || response === true) {
+  if (response === 200) {
```
Comment (jywarren) [inline dist/inlineMarkdownEditor.js]: aha...

**Comment by jywarren** [comment]
I merged this -- do we want to try adjusting tests to protect this or do you think it's pretty straightforward?

**Comment by jywarren** [comment]
Great work, btw!!!

**Pull request merged**